src/OpenMPIR.h

```diff
@@ -276,6 +278,17 @@ class OpenMPDeclareMapperDirective : public OpenMPDirective {
     std::string getUserDefinedIdentifier() { return user_defined_identifier;}
     void setTypeVar (const char * _type_var) { type_var = _type_var;}
     std::string getTypeVar() { return type_var;}
+    void cutTypeAndVar (const char * _type_var) { 
```

Comment:
Open an issue ticket to describe the approach of splitting the type/var string in the lexer, and return the type and var as two individual tokens to the parser. It would be a better solution than this one because we can keep the ompparser grammar the same as OpenMP spec.